binding/lua/Makefile

```diff
@@ -7,7 +7,7 @@ USE_SYSTEM_LUA ?= no
 LUA_VERSION_MAJ_MIN ?= 5.1
 LUA_VERSION ?= $(LUA_VERSION_MAJ_MIN).5
 LUA_VERSION_NOPATCH = $(shell echo -n $(LUA_VERSION) | sed 's!\([0-9]\.[0-9]\).[0-9]!\1!')
-LUA_URL ?= https://github.com/lua/lua/releases/download/$(LUA_VERSION)/lua-$(LUA_VERSION).tar.gz
+LUA_URL ?= https://www.lua.org/ftp/lua-$(LUA_VERSION).tar.gz
```

Comment:
We made a similar change in neovim core: https://github.com/neovim/neovim/pull/5715